packages/explat-client/src/internal/test/state.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import * as State from '../state';
+import { validExperimentAssignmentA, validExperimentAssignmentB } from '../test-common';
+
+describe( 'state', () => {
+	it( 'should save and retrieve valid EAs', () => {
```

Comment:
Let's spell out `EA` or simply use the word "assignment" where it's clear what's meant from the context.